From 31c3384b18dfe9d623b86697f505ccf73efc2ba2 Mon Sep 17 00:00:00 2001
From: hololeap <hololeap@users.noreply.github.com>
Date: Thu, 8 Dec 2022 02:35:51 -0700
Subject: [PATCH] Disable test_package_db test

This tests run 'cabal build' which tries to connect to the web
---

--- hint-0.9.0.7-orig/unit-tests/run-unit-tests.hs	2001-09-09 11:46:40.000000000 +1000
+++ hint-0.9.0.7/unit-tests/run-unit-tests.hs	2023-06-23 09:26:01.434721207 +1000
@@ -520,7 +520,6 @@
 
 ioTests :: [IOTestCase]
 ioTests = [test_signal_handlers
-          ,test_package_db
           ,test_ghc_environment_file
           ]
 
